src/layouts/DarsSideBar/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './DarsSideBar';
```

Comment:
Wait just to clarify, so the right panel is for Dars + CourseDetails?

Reply:
@jason2020 I tried to import CourseDetail into DarsSideBar (so CourseDetail is a component of DarsSideBar); however, CourseDetail needs state props which is also used in CourseGraph. I think this state should be moved to context provider in the future.

Reply:
I just wanted to focus on rearranging files for this pr, so technically this DarsSideBar file is not imported anymore 😟

Reply:
ooo ok so the left panel is dars and right side bar is CourseDetail?

Reply:
i thought the right side bar is dars cuz it's id is "dars"

Reply:
sgsg, yea no worries abt this for now